src/libraries/System.Private.CoreLib/src/System/ComponentModel/DefaultValueAttribute.cs

```diff
@@ -69,7 +74,7 @@ static bool TryConvertFromInvariantString(Type? typeToConvert, string? stringVal
                         Volatile.Write(ref s_convertFromInvariantString, mi == null ? new object() : mi.CreateDelegate(typeof(Func<Type, string, object>)));
                     }
 
-                    if (!(s_convertFromInvariantString is Func<Type?, string?, object> convertFromInvariantString))
+                    if (!(s_convertFromInvariantString is Func<Type, string?, object> convertFromInvariantString))
```

Comment:
Why did we drop the nullable annotation?

Reply:
Because the `ConvertFromInvariantString` method doesn't allow a `null` Type to be passed in, so it was wrong.

https://github.com/dotnet/runtime/blob/82ca681cbac89d813a3ce397e0c665e6c051ed67/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs#L819-L822

`GetConverter` calls `GetDescriptor` and `GetDescriptor` throws if `type` is null.

https://github.com/dotnet/runtime/blob/82ca681cbac89d813a3ce397e0c665e6c051ed67/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs#L902-L912